types/knockout/index.d.ts

```diff
@@ -110,7 +112,7 @@ interface KnockoutObservableStatic {
 }
 
 interface KnockoutObservable<T> extends KnockoutSubscribable<T>, KnockoutObservableFunctions<T> {
-    (): T;
+    <T>(): T;
```

Comment:
interface already declares this type parameter, a new one cannot be used.
Moreover, it does not make sense to redefine type of an observable's underlying type.

Reply:
I do not know how exactly it should be defined, but otherwise example from there https://github.com/DefinitelyTyped/DefinitelyTyped/pull/21569#issuecomment-344900695 doesn't work. These tests were done with IntelliJ IDEA, have not tried with other IDEs.